src/components/nested-sortable/NestedSortable.jsx

```diff
@@ -313,6 +313,7 @@ class NestedSortable extends Component {
             onDragAndDropStart={this.onDragAndDropStart}
             onDragAndDropComplete={this.onDragAndDropComplete}
             isDraggable={item.isDraggable}
+            doesAllowDrop={item.doesAllowDrop}
```

Comment:
I'd prefer just `allowDrop`

Reply:
I was thinking in the same way as you, but as we already have doesAllowDropIntoRoot, I kept the same logic, I can change it to allowsDrop if you want.